techmatters/flex-plugins
# CHI-1406: Fix accessibility for CSAM dropdown & add clickaway dismiss

## Description

* Use HTML button tag for CSAM type dropdown for correct accessibility behaviour
* Add clickaway dismissal to CSAM type dropdown

### Checklist
- N/A New tests added

### Related Issues
Fixes CHI-1406

### Verification steps

* See ticket comments
* Also verify keyboard interactions with CSAM type dropdown are as expected
